<launch>
	<arg name="joy" default="/dev/input/js0"/>
	<arg name="joy_deadzone" default="0.02"/>
	<arg name="joy_topic" default="/input/joy_data" />

	<node name="joy_node" pkg="joy" exec="joy_node" respawn="true" output="screen">
		<param name="dev" value="$(var joy)" />
		<param name="deadzone" value="$(var joy_deadzone)" />
		<param name="autorepeat_rate" value="100.0" />
		<param name="coalesce_interval" value="0.005" />
	</node>

	<node name="joy_adapter" pkg="m2_joy" exec="joy_transform_data_node" respawn="true" output="screen">
		<param name="joy_topic" value="$(var joy_topic)" />
	</node>
</launch>
